includes/pages/admin_free.php

```diff
@@ -117,7 +117,7 @@ function admin_free()
             div('row', [
                 div('col-md-12 form-inline', [
                     div('row', [
-                        form_text('search', __('form.search'), $search, null, null, null, 'col'),
+                        form_text('search', __('form.search'), $search, null, false, null, null, null, ['col']),
```

Comment:
this is a bit of a drive-by fix: when I checked all `form_text()` calls, I noticed that this call was missing an another argument in addition to the newly introduced one and that the last argument had a type mismatch. So I updated that on the go – the page looks fine in my browser, but that's the full extent of my testing. so, happy to reverse if desired.